### DISCUSSION TITLE
Support offline build

### DISCUSSION THREAD

**Discussion by gwankyun** [fltk-rs/fltk-rs#1404] [Q&A]
**Is your feature request related to a problem? Please describe.**

I want to use the library offline

```shell
cargo vender
```

and copy the project directory to offline environment and run:

```shell
cargo build --offline
```

it repored error, since it need to download lib_x64-windows-msvc.tar.gz。

**Describe the solution you'd like**

bundled.rs can use the local gz file.

**Describe alternatives you've considered**

**Additional context**

**Comment by MoAlyousef** [comment]
Vendoring and offline builds works. 

If you are using the fltk-bundled feature, you can download the bundle, unpack it somewhere and use the CFLTK_BUNDLE_DIR environment variable to point to it.
